docs/xdr/features/collect/integrations/cloud_and_saas/salesforce.md

```diff
@@ -0,0 +1,33 @@
+uuid: 2b13307b-7439-4973-900a-2b58303cac90
+name: Salesforce
+type: intake
+
+## Overview
+
+Salesforce provides customer relationship management software and applications focused on sales, customer service, marketing automation, e-commerce, analytics, and application development.
+
+{!_shared_content/operations_center/detection/generated/suggested_rules_2b13307b-7439-4973-900a-2b58303cac90_do_not_edit_manually.md!}
+
+{!_shared_content/operations_center/integrations/generated/2b13307b-7439-4973-900a-2b58303cac90.md!}
+
+## Configure
+
+This setup guide will show you how to provide an integration between Salesforce events and SEKOIA.IO.
+
+### Prerequisites
+
+- Have an internal concentrator (Salesforce)
```

Comment:
Internal concentrator is only required for collect through syslog.
As we have a connector, here, we don't need a concentrator.
```suggestion
```